src/config/plugin.js

```diff
@@ -11,7 +11,7 @@ function getPackagePlugins(cwd) {
   return [
     ...Object.keys(pkg.dependencies || {}),
     ...Object.keys(pkg.devDependencies || {}),
-  ].filter(dep => /^nwb-/.test(dep))
+  ].filter(dep => /^nwb-/.test(dep) || /^@[^/]+\/nwb-/.test(dep))
```

Comment:
Suggest making the scope an optional part of the same `RegExp` instead:

`/^(@[^/]+\/)?nwb-/`